src/main/java/uk/gov/hmcts/ccd/auditlog/AuditRepository.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.ccd.auditlog;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AuditRepository {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AuditRepository.class);
+
+    public void save(final AuditEntry auditEntry) {
+        LOG.info(auditEntry.toString());
```

Comment:
I think the AuditService could be injected with a AuditLogFormatter which is responsible for formatting the audit string. Then the Repository just logs the formatted entry

Reply:
if you introduce formatter class then it should inject here rather than AuditServcie.  I like passing domain entity to repository